repository/e.json

```diff
@@ -629,6 +624,18 @@
 				}
 			]
 		},
+		{
+			"name": "Elm Syntax Highlighting",
+			"details": "https://github.com/evancz/elm-syntax-highlighting",
+			"labels": ["elm", "language syntax", "highlighting"],
+			"previous_names": ["Elm Language Support"],
```

Comment:
Since you effectively removed functionality from the package, you should add an update message that informs people of the changes and what other package to get. See https://packagecontrol.io/docs/messaging for details.

Reply:
Good point! I added this documentation in https://github.com/evancz/elm-syntax-highlighting/commit/6486db6b10e0ff45edbcbee530268b1eaf17fd48